public/locales/mn/translation.json

```diff
@@ -0,0 +1,45 @@
+{
+  "website": {
+    "name": "Азиудын эсрэг үзэн ядалтаас үүдсэн гэмт хэргийн бүртгэл"
+  },
+  "location" : "Байршил",
+  "date_range" : "Огнооны муж",
+  "daily_count" : "Өдөр бүрийн тоо",
+  "monthly_count" : "Сар бүрийн тоо",
+  "load_more" : "Үргэлжлүүлэх",
+  "incident_count_by_state" : "Хэргийн тоо аймгаар",
+  "incident_table" : {
+    "total_incident" : "Нийт {{count}} тохиолдол",
+    "no_data" : "Мэдээлэл байхгүй",
+    "total_incident_plural" : "Нийт {{count}} тохиолдлууд",
+    "state" : "Аймаг",
+    "count" : "Тоо",
+    "count_10k_asian" : "Тоо/10K Азиуд",
+    "count_1mm" : "Тоо/1MM"
+  },
+  "incident_map" : {
+    "cases" : "Хэргүүд",
+    "no_data" : "Мэдээлэл байхгүй",
+    "count_10k_asian" : "Тоо/10K Азиуд",
+    "count_1mm" : "Тоо/1MM"
+  },
+  "incident_chart" : {
+    "total_monthly_cases" : "Сард нийт {{count}} хэрэг",
+    "total_monthly_cases_plural" : "Сард нийт {{count}} хэргүүд",
+    "total_daily_cases" : "Өдөрт нийт {{count}} хэрэг",
+    "total_daily_cases_plural" : "Өдөрт нийт {{count}} хэргүүд",
+    "trend" : "Үзэн ядалтаас үүдсэн гэмт хэргийн төлөв",
+    "no_data" : "Мэдээлэл байхгүй",
+    "total_cases" : "Нийт {{count}} хэрэг",
+    "total_cases_plural" : "Нийт {{count}} хэргүүд"
+  },
+  "contact_us" : "Холбоо барих",
+  "copyright" : "Зохиогчийн эрх",
+  "disclaimer" : {
+    "title" : "Мэдэгдэл",
+    "here" : "Энд",
+    "1" : "Азиудын эсрэг үзэн ядалтаас үүдсэн гэмт хэргийн бүртгэл вэб хуудсыг сайн дурын бүлэг бүтээн ажиллуулж буй бөгөөд, өдөр тутам тохиолдож буй Азиудыг гадуурхах гэмт хэргийг нийгэмд таниулах цор ганц зорилготой юм.",
+    "2" : "Бид Азиудыг гадуурхах хэргийн мэдээллийг нээлттэй эх сурвалжуудаас цуглуулан анхны мэдээлэлрүү хүргэх холбоосоор хангадаг (гарчиг болон товч агуулгын хамтаар). Бид 17 U.S.C. § 107 заалтын дагуу аливаа зохиогчийн эрхтэй материалууд нь “fair use” хэрэглээний хүрээнд зөвшөөрөгдсөн гэж үзэж байна. Хэрэв танд ямар нэгэн асуулт болон санал хүсэлт байвал <1>энд</1> холбоо барина уу.",
```

Comment:
The file needs to be well formatted JSON. Can you escape the " signs? Thanks!

Reply:
Sorry, missed that fixed it. And validated it, looks alright now.